modules/channel-web/src/views/lite/translations/index.tsx

```diff
@@ -1,18 +1,38 @@
+import fromPairs from 'lodash/fromPairs'
 import { addLocaleData } from 'react-intl'
 
+import localeAr from 'react-intl/locale-data/ar'
 import localeEn from 'react-intl/locale-data/en'
+import localeEs from 'react-intl/locale-data/es'
 import localeFr from 'react-intl/locale-data/fr'
 import localePt from 'react-intl/locale-data/pt'
-import localeEs from 'react-intl/locale-data/es'
+import localeRu from 'react-intl/locale-data/ru'
+import localeUk from 'react-intl/locale-data/uk'
 
+import ar from './ar.json'
 import en from './en.json'
+import es from './es.json'
 import fr from './fr.json'
 import pt from './pt.json'
-import es from './es.json'
+import ru from './ru.json'
+import uk from './uk.json'
 
-const availableLocale = ['en', 'fr', 'pt', 'es']
+const availableLocale = ['en', 'fr', 'pt', 'es', 'ar', 'ru', 'uk']
 const defaultLocale = 'en'
-const translations = { 'en-US': en, en, fr, pt, es }
+
+const arLocales = ['ar', 'ar-SA', 'ar-IQ', 'ar-EG', 'ar-LY', 'ar-DZ', 'ar-MA', 'ar-TN', 'ar-OM', 'ar-YE',
```

Comment:
quick question regarding this. Instead of providing every locales variations, could we simply parse the browser's local match `ar` only and discard the trailings ?

Reply:
I can do that, but can there be different wordings for different wordings?

Reply:
I think that's alright as-is for now – does this impact the webchat bundle size considerably btw?

Reply:
Seems to add barely 5kb to the final bundle size.

Reply:
Refactored those region subtags - seems to look cleaner now. :smiley_cat: